services/venice-controller/src/test/java/com/linkedin/venice/controller/TestVeniceParentHelixAdmin.java

```diff
@@ -301,7 +299,7 @@ public void testAddStore() {
     when(zkClient.readData(zkMetadataNodePath, null)).thenReturn(null)
         .thenReturn(
             AdminTopicMetadataAccessor
-                .generateMetadataMap(Optional.of(1L), Optional.of(-1L), Optional.of(1L), Optional.empty()));
+                .generateMetadataMap(Optional.of(1L), Optional.of(-1L), Optional.of(1L), Optional.of(84L)));
```

Comment:
Let's replace the hardcode version number to the latest version number in AvroProtocolDefinition.

Reply:
Updated.